### PR TITLE
[compiler-v2] Addressing open issues of #11519

### PR DESCRIPTION
**Pull request by wrwg**

### CODE REVIEW

**Review by rahxephon89 — APPROVED** [review]
LGTM, thanks!

File: third_party/move/move-compiler-v2/src/file_format_generator/function_generator.rs [hunk 1]
```diff
@@ -765,6 +739,42 @@ impl<'a> FunctionGenerator<'a> {
...
+        // Notice that we flush the stack _before_ the next processed instruction, therefore
+        // we use the before version of the below function.
+        self.abstract_flush_stack_before(ctx, stack_to_flush)
+    }
+
+    /// Determines the maximal prefix of `temps` which are already on the stack, and
+    /// returns the temps which are not and need to be pushed.
+    fn analyze_stack<'t>(
+        &mut self,
+        _ctx: &BytecodeContext,
```
Comment (rahxephon89) [inline third_party/move/move-compiler-v2/src/file_format_generator/function_generator.rs:763]: Why do we need this argument in this function?

Comment (brmataptos) [reply third_party/move/move-compiler-v2/src/file_format_generator/function_generator.rs:763]: +1

**Review by brmataptos — APPROVED** [review]
Looks good aside from the unused parameter Teng points out.

File: third_party/move/move-compiler-v2/src/file_format_generator/function_generator.rs (hunk 1 above)
Comment (wrwg) [reply third_party/move/move-compiler-v2/src/file_format_generator/function_generator.rs]: Done

**Pull request merged**